benchmarks.md

```diff
@@ -0,0 +1,115 @@
+# Benchmark Results
+
+All numbers reported here refer to cycle counts.
```

Comment:
Can you add the speed somewhere so that it's possible to convert from cycles to time?

Reply:
Yes, I see I should also update some of the Readmes.